lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -3010,6 +3023,15 @@ def _handle_array_type(attype):
                 "Did not recognize type '%s' of column '%s'" % (attype, name)
             )
             coltype = sqltypes.NULLTYPE
+
+        # If a zero byte (''), then not a generated column.
+        #  Otherwise, s = stored. (Other values might be added in the future.)
+        if generated != "":
```

Comment:
OK it comes back as blank....OK None / blank would both test here for just "if generated"

Reply:
ok i'll change it